pvh/pvhboot.go

```diff
@@ -133,8 +133,10 @@ func (h *HVMMemMapTableEntry) Bytes() ([]byte, error) {
 	return buf.Bytes(), nil
 }
 
-func CreateGDT() [4]uint64 {
-	var gdtTable [4]uint64
+type GDT [4]uint64
+
+func CreateGDT() GDT {
+	var gdtTable GDT
```

Comment:
Not sure if we need a struct for GDT.
Sure, Byte function is cleaner, but we use it in one instance only once.

Also GDT can have more than 4 entries. :) 

I leave it to you

Reply:
Thanks for the review!

As you said, it's only used once now, but Bytes() functions are implemented the same meaning for various data structures and I find it easy to read.
If we need to make the number of entries variable length, we will handle that in a future PR.